### PR TITLE
Main Page & Report Form Updates

### PR DESCRIPTION
**Pull request by sudo-joseph**
This PR updates the look & feel of the main page and add new form. Adds new feature to link report in sidebar to map pin. (click link to show associated location). Updated Report model with updated fields. Updated form to utilize new fields. 

Known Issues: Image upload no longer seems to be working with form changes. need to investigate further.

### CODE REVIEW

**Comment by sudo-joseph** [comment]
Wait on reviewing till i fix a bug on the main page where map is randomly moving on link click. was working earlier :(

**Comment by sudo-joseph** [comment]
This is ready to merge. Please review.

File: apps/core/views.py [hunk 1]
```diff
@@ -7,12 +7,21 @@
 import json
 from geopy.geocoders import MapBox
 import os
+import pytz
```
Comment (ishinan) [inline apps/core/views.py:10]: Where did you use this???

**Review by ishinan — APPROVED** [review]

File: apps/core/views.py (hunk 1 above)
Comment (sudo-joseph) [reply apps/core/views.py:10]: It's part of the datetime stuff. not 100% sure if its being used. django recommends installing it https://docs.djangoproject.com/en/3.0/topics/i18n/timezones/

Comment (sudo-joseph) [reply apps/core/views.py:10]: removed. will test locally

Comment (sudo-joseph) [reply apps/core/views.py:10]: seems to be needed and branch is working correctly. going to merge.

**Comment by sudo-joseph** [comment]
Conflicts resolved. Going to merge into master.

**Comment by ishinan** [comment]
Thank you so much, Joseph

**Pull request merged**